intezeranalyze_connector.py

```diff
@@ -124,6 +128,19 @@ def _make_rest_call(self, endpoint, action_result, headers=None, params=None, da
         # Create a URL to connect to
         url = self._base_url + endpoint
 
+        if endpoint != 'is-available' and not self._jwt_token:
+            response = requests.post(self._base_url + 'v2-0/get-access-token', json={'api_key': config.get('apiKey')})
+            if response.status_code != HTTPStatus.OK:
+                return RetVal(action_result.set_status(phantom.APP_ERROR,
+                                                       "Error Connecting to server. Details: {0}".format(str(response.text))),
+                              resp_json)
+            self._jwt_token = response.json()['result']
```

Comment:
i'm not sure how this is implemented in phantom, but jwt can expire if this class is up for more than 10 hours. catch 401 responses and retry one more time after get access token